src/GovUk.Education.ExploreEducationStatistics.Admin.Tests/Services/ThemeServiceTests.cs

```diff
@@ -485,7 +485,71 @@ public async Task DeleteTheme_OtherThemesUnaffected()
                 VerifyAllMocks(topicService);
 
                 result.AssertRight();
-                Assert.Equal(otherTheme.Id, context.Themes.AsQueryable().Select(t => t.Id).Single());
+                Assert.Equal(otherTheme.Id, await context.Themes.AsQueryable().Select(t => t.Id).SingleAsync());
+            }
+        }
+
+        [Fact]
+        public async Task DeleteUITestThemes()
```

Comment:
I'd create an additional test `DeleteUITestThemes_DisallowedByConfiguration` - like the test for `DeleteTheme`.